.github/workflows/CI.yml

```diff
@@ -9,14 +9,12 @@ on:
 
 jobs:
   spotless:
-    if: github.repository == 'opensearch-project/opensearch-ai-flow-framework'
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - name: Spotless Check
         run: ./gradlew spotlessCheck
   javadoc:
-    if: github.repository == 'opensearch-project/opensearch-ai-flow-framework'
```

Comment:
We should have the repo check or else it will run on the fork as before?

Reply:
> We should have the repo check or else it will run on the fork as before?

I specifically removed it as I think it's a good thing to run spotless and javadoc on anyone's branch before they submit a PR.

Reply:
Looking at the number of emails we receive from all the repos. I would suggest to have this check please :)

Reply:
You don't like lots of emails?
![Screenshot 2023-10-16 at 10 30 39 AM](https://github.com/opensearch-project/opensearch-ai-flow-framework/assets/9291703/609a94fa-83ff-4ce9-b6a1-ac477ff6fa44)

Reply:
My thought was that this will only generate an email if it fails, and if it fails on your branch before the PR sending the submitter an email it's better than failing on the PR and sending all the maintainers an email.  But I'll go ahead and put it back.

Reply:
How many labels more I need now? :(
![Screenshot 2023-10-16 at 10 34 29 AM](https://github.com/opensearch-project/opensearch-ai-flow-framework/assets/16099877/2d0d0d9d-367f-4706-a282-b89cb8d80bcf)